[2025-02-17T09:05:51.899+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-02-17T09:05:51.922+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_trino_dag.print_command manual__2025-02-17T09:05:50.030854+00:00 [queued]>
[2025-02-17T09:05:51.931+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_trino_dag.print_command manual__2025-02-17T09:05:50.030854+00:00 [queued]>
[2025-02-17T09:05:51.931+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-02-17T09:05:51.944+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): print_command> on 2025-02-17 09:05:50.030854+00:00
[2025-02-17T09:05:51.952+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=2268) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-17T09:05:51.954+0000] {standard_task_runner.py:63} INFO - Started process 2269 to run task
[2025-02-17T09:05:51.953+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'my_first_trino_dag', 'print_command', 'manual__2025-02-17T09:05:50.030854+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/my_first_trino_dag.py', '--cfg-path', '/tmp/tmpk2pnevi3']
[2025-02-17T09:05:51.954+0000] {standard_task_runner.py:91} INFO - Job 24: Subtask print_command
[2025-02-17T09:05:52.007+0000] {task_command.py:426} INFO - Running <TaskInstance: my_first_trino_dag.print_command manual__2025-02-17T09:05:50.030854+00:00 [running]> on host 800d9ac51cc2
[2025-02-17T09:05:52.103+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_first_trino_dag' AIRFLOW_CTX_TASK_ID='print_command' AIRFLOW_CTX_EXECUTION_DATE='2025-02-17T09:05:50.030854+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-17T09:05:50.030854+00:00'
[2025-02-17T09:05:52.103+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-02-17T09:05:52.127+0000] {logging_mixin.py:188} INFO - Return Count:  [5332737]
[2025-02-17T09:05:52.127+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-02-17T09:05:52.127+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-02-17T09:05:52.136+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=my_first_trino_dag, task_id=print_command, run_id=manual__2025-02-17T09:05:50.030854+00:00, execution_date=20250217T090550, start_date=20250217T090551, end_date=20250217T090552
[2025-02-17T09:05:52.168+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-02-17T09:05:52.188+0000] {taskinstance.py:3503} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2025-02-17T09:05:52.190+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
